autoload/dirvish.vim

```diff
@@ -388,6 +388,7 @@ function! dirvish#open(...) range abort
     return
   endif
   if !&hidden && &modified
+      \ && (!exists("*win_findbuf") || len(win_findbuf(winbufnr(0))) == 1)
```

Comment:
It just seemed appropriate since this is explicitly concerned with windows.  I guess they should get the same value.